themes/custom_theme/valencia-theme/src/templates/portal_normal.ftl

```diff
@@ -0,0 +1,115 @@
+
+<@liferay_ui["quick-access"] contentId="#main-content" />
+
+<@liferay_util["include"] page=body_top_include />
+
+<@liferay.control_menu />
+
+<div class="container-fluid liferay-menu-buffer" id="wrapper">
+	<header id="banner" role="banner">
+		<#if !is_signed_in && !has_navigation>
+			<a data-redirect="${is_login_redirect_required?string}" href="${sign_in_url}" id="sign-in" rel="nofollow">${sign_in_text}</a>
+		</#if>
+
+		<#if has_navigation && is_setup_complete>
+			<#include "${full_templates_path}/navigation.ftl" />
+		</#if>
+		<div id="header-carousel" class="carousel slide" data-ride="carousel">
+			<div class="carousel slide" data-ride="carousel">
+				<div class="carousel-inner">
+					<div class="item active">
+						<div class="page-header header-filter" style="background-image: url('http://i.imgur.com/nsHGXAf.jpg');">
+							<div class="container">
+								<div class="row">
+									<div class="col-md-8 col-md-offset-2 text-center">
+										<h1 class="site-title">
+											<a class="${logo_css_class}" href="${site_default_url}" title="<@liferay.language_format arguments="${site_name}" key="go-to-x" />"><img alt="${logo_description}" height="${site_logo_height}" src="${site_logo}" width="${site_logo_width}" /></a>
+											
+											<#if show_site_name>
+												<span class="site-name" title="<@liferay.language_format arguments="${site_name}" key="go-to-x" />">
+												${site_name}
+												</span>
+											</#if>
+										</h1>
+										<h2>Helping you go farther than you imagined.</h2>
+										<div class="buttons">
+											<a href="#" class="btn btn-primary btn-lg">See more</a>
+										</div>
+									</div>
+								</div>
+							</div>
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>		
+	</header>
+
+	<div class="main main-raised">
+			<section class="section-wrapper">
+					<div class="container">
+							<div class="row">
+									<div class="col-md-8 col-md-offset-2">
+											<h5 class="description">This is the paragraph where you can write more details about your product. Keep you user engaged by providing
+													meaningful information. Remember that by this time, the user is curious, otherwise he wouldn&#039;t
+													scroll to get here.</h5>
+									</div>
+							</div>
+							<div class="row">
+								<div class="col-xs-12">
+									<section id="content">
+										<h1 class="hide-accessible">${the_title}</h1>
+
+										<nav id="breadcrumbs">
+											<@liferay.breadcrumbs />
+										</nav>
+
+										<#if selectable>
+											<@liferay_util["include"] page=content_include />
+										<#else>
+											${portletDisplay.recycle()}
+
+											${portletDisplay.setTitle(the_title)}
+
+											<@liferay_theme["wrap-portlet"] page="portlet.ftl">
+												<@liferay_util["include"] page=content_include />
+											</@>
+										</#if>
+
+									</section>
+
+								</div>
+							</div>
```

Comment:
It looks like you are using some Bootstrap column layouts.  You should probably create a Liferay theme layout that will work for your theme.  

See https://dev.liferay.com/develop/tutorials/-/knowledge_base/7-0/creating-layout-templates-with-the-themes-generator-0 and https://dev.liferay.com/develop/tutorials/-/knowledge_base/7-0/creating-layout-templates-manually for more details.